codalab/lib/bundle_cli.py

```diff
@@ -2952,7 +2952,7 @@ def do_bs_health_check(self, args):
 
     def _fail_if_headless(self, args):
         if self.headless:
-            raise UsageError('Cannot execute CLI command: %s' % args.command)
+            raise UsageError('Cannot execute CLI command "%s" because you are using a headless CLI.' % args.command)
```

Comment:
This will make no sense from the web browser.  Should say: You are only allowed to execute command "%s" from the CLI.